README.md

```diff
@@ -374,3 +374,28 @@ In Python:
 ```
 python py/htm/examples/mnist.py
 ```
+
+## Cite us
+
+We're happy that you can use the community work in this repository or even join the development! 
+Please give us attribution by [linking to us as htm.core](https://github.com/htm-community/htm.core/), 
```

Comment:
nit:  I think you should actually show the link rather than have words mapped to the link. If someone is looking for something to copy/paste the link should be showing.
i.e.  Rather than:
```
[linking to us as htm.core](https://github.com/htm-community/htm.core/)
```
use something like:   linking to us as [htm.core](https://github.com/htm-community/htm.core/) at https://github.com/htm-community/htm.core/
```
"linking to us as [htm.core](https://github.com/htm-community/htm.core/) at https://github.com/htm-community/htm.core/"
```

Reply:
Good point, added as you suggest. Thanks & please re-review